resources/patches/graphql+17.0.0-alpha.5.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/node_modules/graphql/execution/execute.js b/node_modules/graphql/execution/execute.js
```

Comment:
@dimaMachina again, a reminder that this patch is being published to users. we should not be patching `graphql` before publishing as a reference ecosystem

Reply:
This patch was not published to users